include/mbedtls/ssl.h

```diff
@@ -2533,13 +2533,19 @@ void mbedtls_ssl_conf_cert_req_ca_list( mbedtls_ssl_config *conf,
 #if defined(MBEDTLS_SSL_MAX_FRAGMENT_LENGTH)
 /**
  * \brief          Set the maximum fragment length to emit and/or negotiate
- *                 (Default: the smaller of MBEDTLS_SSL_IN_CONTENT_LEN and
+ *                 (Typical: the smaller of MBEDTLS_SSL_IN_CONTENT_LEN and
```

Comment:
Minor: Please use `#` to have Doxygen generate a link to the documentation of `MBEDTLS_SSL_IN_CONTENT_LEN`.